docs/usage/self-hosted-configuration.md

```diff
@@ -755,6 +755,39 @@ If left as default (null), a random short ID will be selected.
 
 ## logFileLevel
 
+## mergeConfidenceDatasources
+
+This feature is applicable only if you have an access token for Mend's Merge Confidence API.
+
+If set, Renovate will query the merge-confidence JSON API only for datasources that are part of this list.
+Otherwise, it queries all the supported datasources (check default value).
+
+Example:
+
+```js
+modules.exports = {
+  mergeConfidenceDatasources: ['npm'],
+};
+```
+
+## mergeConfidenceEndpoint
+
+This feature is applicable only if you have an access token for Mend's Merge Confidence API.
+
+If set, Renovate will retrieve Merge Confidence data by querying this API.
+Otherwise, it will use the default URL, which is <https://developer.mend.io/>.
+
+If you use the Mend Renovate Enterprise Edition (Renovate EE) and:
+
+- have a static merge confidence token that you set via `MEND_RNV_MC_TOKEN`
+- _or_ set `MEND_RNV_MC_TOKEN` to `auto`
+
+Then you must set this variable at the _server_ and the _workers_.
```

Comment:
This means we need to update `CE/EE` to use the same naming for consistency as we currently expect `RENOVATE_X_...` for overriding the api base url.

Reply:
CE/EE?

Reply:
@RahulGautamSingh I believe this comment for me and you can ignore/resolve it
CE: Community Edition
EE: Enterprise Edition

@Gabriel-Ladzaretti `RENOVATE_X_MERGE_CONFIDENCE_API_BASE_URL` will still work

Reply:
we'll need to update this documentation after implementing better handling in EE (no more tokens in the worker container)